docker/Gruntfile.patch

```diff
@@ -1,6 +1,17 @@
---- Gruntfile.js	2016-12-22 10:02:55.763233997 -0500
-+++ new.js	2016-12-22 10:02:00.083913992 -0500
-@@ -112,6 +112,10 @@
+diff --git a/Gruntfile.js b/Gruntfile.js
+index 87630d5..687d5cd 100644
+--- a/Gruntfile.js
++++ b/Gruntfile.js
+@@ -101,7 +101,7 @@ module.exports = function(grunt) {
+ 		},
+ 
+ 		zip: {
+-			'reveal-js-presentation.zip': [
++			'/reveal-js-presentation.zip': [
```

Comment:
1. I wish there was a better way to represent a diff of a diff
2. I do this because currently I'm not sure how one writes to a union-fs, or if there's a way to set up the union to say "writes go to this directory"